python/lsst/ip/isr/crosstalk.py

```diff
@@ -53,19 +55,23 @@ class CrosstalkCalib(IsrCalib):
     _SCHEMA = 'Gen3 Crosstalk'
     _VERSION = 1.0
 
-    def __init__(self, detector=None, **kwargs):
+    def __init__(self, detector=None, nAmp=0, **kwargs):
         self.hasCrosstalk = False
-        self.nAmp = 0
+        self.nAmp = nAmp if nAmp else 0
         self.crosstalkShape = (self.nAmp, self.nAmp)
 
-        self.coeffs = None
-        self.coeffErr = None
-        self.coeffNum = None
-        self.interChip = None
+        self.coeffs = np.zeros(self.crosstalkShape) if self.nAmp else None
+        self.coeffErr = np.zeros(self.crosstalkShape) if self.nAmp else None
+        self.coeffNum = np.zeros(self.crosstalkShape,
```

Comment:
Could you briefly just say what this actually is? Is it the number of points that went into making a given value?

Reply:
I've added a full description in the class docstring, which almost certainly should be duplicated in other calibration types.